plots.py

```diff
@@ -283,6 +273,23 @@ def indiv_derived_props_plots(fitspath, dataset, restrict_MTO = False, revised =
     else:    
         mass_metal_tbl = asc.read(fitspath + 'massbin/' + dataset + filename_dict['bin_derived_prop'])
         LHb_metal_tbl = asc.read(fitspath + 'mass_LHbeta_bin/' + dataset + filename_dict['bin_derived_prop'])
+        
+    if err_bars == True:
+        der_prop_err_M = np.load(fitspath + 'massbin/' + dataset + npz_filename_dict['der_prop_errors'])
+        der_prop_err_LHb = np.load(fitspath + 'massbin/' + dataset + npz_filename_dict['der_prop_errors'])
```

Comment:
`os.path.join` would make this easier as it accepts multiple folders as separate inputs:
`join(fitspath, 'massbin', dataset, npz_filename_dict['der_prop_errors'])`